README.md

```diff
@@ -15,6 +15,8 @@ The following folders can be found in this repository:
 - **[`metriccentury`](https://github.com/deephaven/examples/tree/main/metriccentury)** - Data recorded from a 100 km bike ride
 - **[`pems`](https://pems.dot.ca.gov/)** - Traffic flow data collected near Davis, CA.
 - **[`taxi`](https://azure.microsoft.com/en-us/services/open-datasets/catalog/nyc-taxi-limousine-commission-yellow-taxi-trip-records/)** - Yellow Taxi trip records
+- **[`fit`](https://www.strava.com/) - Workout results in the proprietary fit format developed by Garmin. Downloadable from e.g. Strava
+- **[`tickingHeartRate`](https://deephaven.io/core/docs/how-to-guides/dynamic-table-writer) - Simulated ticking heart rate data
```

Comment:
```suggestion
- **[`tickingHeartRate`](https://deephaven.io/core/docs/how-to-guides/dynamic-table-writer) - Simulated ticking heart rate data.
```

Reply:
The link is throwing me off. Our DynamicTableWriter does not have this table.

Reply:
it doesn't... I'm not really sure what to link to / where to link, because this is a self-generated example using our own internal tooling. Can I have no link?